chef/cookbooks/hyperv/recipes/windows_features.rb

```diff
@@ -15,6 +15,7 @@
   node[:features_list][:windows].each do |feature_list|
     windows_feature feature_list do
       action :install
+      all true
```

Comment:
I'm confused: I don't see the `all` attribute being defined in https://github.com/crowbar/barclamp-hyperv-compute/blob/master/chef/cookbooks/windows/resources/feature.rb. How does this work?

Also, I thought this had to happen earlier? Is that not the case anymore?